### PR TITLE
Support Windows paths

### PR DESCRIPTION
**Pull request by pdugas**

### CODE REVIEW

**Comment by pdugas** [comment]
The only changes in the test should be to make the Bucket value a valid path on the host machine using the local separator.

File: lib/mock.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ function walk (dir) {
 	var list = fs.readdirSync(dir)
 
 	list.forEach(function (file) {
-		file = dir + '/' + file
+		file = dir + path.sep + file
```
Comment (spacer59) [inline lib/mock.js:26]: nit: using `join` makes it more readable, applies to all occurrences here

Comment (spacer59) [reply lib/mock.js:26]: fine with keeping it as is also, this PR looks good

**Review by spacer59 — APPROVED** [review]
lgtm, curious why we decided to fork instead of branch. It looks like we can use a branch sha in `package.json`

File: lib/mock.js (hunk 1 above)
Comment (pdugas) [reply lib/mock.js:26]: Gonna leave it as is since it's closer to the original.

**Comment by pdugas** [comment]
I didn't have branch rights in this PR originally. Do now :)

**Pull request merged**